lib/resque.rb

```diff
@@ -187,6 +187,46 @@ def enqueue_front
     end
   end
 
+  attr_writer :per_worker_stats
+  def per_worker_stats
+    if defined? @per_worker_stats
+      @per_worker_stats
+    else
+      @per_worker_stats = true
+    end
+  end
+
+  attr_writer :track_starts
+  def track_starts
+    if defined? @track_starts
+      @track_starts
+    else
+      @track_starts = true
+    end
+  end
+
+  attr_writer :queues_in_names
+  def queues_in_names
+    if defined? @queues_in_names
+      @queues_in_names
+    else
+      @queues_in_names = true
+    end
+  end
+
+  attr_writer :queue_name_prefix
+  def queue_name_prefix(&block)
+    if block
+      @queue_name_prefix = block
+    elsif defined? @queue_name_prefix
+      @queue_name_prefix
+    else
+      @queue_name_prefix = -> (x) { x }
+    end
+  end
```

Comment:
FEATURE: `queue_name_prefix`

A hook to decorate queue names passed to `Resque.enqueue_to`. Uses a Proc because the monolith doesn't prefix FS queues.

Could be PRed to master. Needs some caution around mixing APIs, however - code should use methods like `enqueue_to` rather than raw `push`.